src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging.Extraction.csproj

```
@@ -0,0 +1,360 @@
+﻿<Project>
+  <PropertyGroup>
+    <RequiresSigningXplatAPIs>true</RequiresSigningXplatAPIs>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <Description>NuGet's understanding of packages. Reading nuspec, nupkgs and package signing.</Description>
+    <TargetFrameworks>net45</TargetFrameworks>
```

Avoid `net45` TFM in source-build mode:

```suggestion
    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">$(TargetFrameworks);net45</TargetFrameworks>
```

---

will test this... think i want this to only be net45...
is there another way to make source build ignore this project all together?
does build.ps1 end up building it because it is part of the sln?

---

made this change...but not sure i want to keep it. keeping conversation open until i test a few things.

---

build failures locally and on CI. could likely fix...but want to figure out best direction here.

how can i test source build locally? does build.ps1 have a switch to turn it on?

---

talking with @dominoFire, decided to set xplatproject to false, which will keep it out of restorexplat and buildxplat targets

---

fixed in next iteration